### ISSUE TITLE
Nan: Predicted Value

### ISSUE THREAD

**Issue by cherylLbt** [FlorentF9/DeepTemporalClustering#13]
Hi,
I got all nan predicted values when running your code. In DTC.fit, everything goes well when calculating p and q for the first time just after "init_cluster_weights". But when it reached "model.fit(X_train, [X_train, p]" for the first time and then got predicted value in the next iteration, all predicted values turned to be nan.

I have tried several ways to modify it, including:
enlarge batchsize
reduce learning rate of Adam
grad clip
redefine the KL loss function to avoid log(0)

I am sure there is no nan or inf in the input data. Could you please help me to solve the problem?

Looking forward to your reply and HAPPY CHRISTMAS HOLIDAY!

**Comment by FlorentF9** [comment]
Hi, 

Thanks, and best wishes for this new year! (including making this code work)
First, are you using one of the UCR benchmark data sets, or your own data set?

**Comment by cherylLbt** [comment]
I only use my own data set. The strange thing is that I happened to use another computer and the problem disappeared. 

I think there may be some exclusive bugs in the tensorflow. Because I didn't change any codes, the only difference between these two computers is that they have different versions of tensorflow.

Thanks for your reply!
